### PR TITLE
invoke task to make bcrypt releases a bit simpler

### PR DESCRIPTION
**Pull request by reaperhulk**
This will tag the release in github, upload the sdist, purge the PyPI cache, trigger the wheel builder, and download/upload the wheels.

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        "twine upload -s dist/bcrypt-{0}*".format(version)
+    )
+
+    session = requests.Session()
+
+    # This tells the CDN to delete the cached response for the URL. We do this
+    # so that the Jenkins builders will see the new sdist immediately when they
+    # go to build the wheels.
+    response = session.request(
+        "PURGE", "https://pypi.python.org/simple/bcrypt/"
```
Comment (dstufft) [inline tasks.py]: Hmm, why is this required? Uploading should already do this.

Comment (reaperhulk) [inline tasks.py]: It probably isn't! I just cargo-culted it over from cryptography, so maybe we should remove it there too. I'll remove it here.
